package.json

```diff
@@ -39,10 +39,10 @@
     "require-from-string": "^2.0.2"
   },
   "peerDependencies": {
-    "@mediamonks/eslint-config-base": "^2.0.0-alpha.4",
-    "@mediamonks/monck": "^1.0.1",
+    "@muban/eslint-config": "^1.0.0-alpha.5",
+    "@mediamonks/monck": "^1.0.6",
     "@muban/muban": "^1.0.0-alpha.28",
-    "@muban/template": "^1.0.0-alpha.1",
+    "@muban/template": "^1.0.0-alpha.3",
```

Comment:
after updating/changing dependencies, you should always run `npm install` so the `package-lock.json` is updated with the resolved versions.